_rules/printable-characters-shortcut-ffbc54.md

```diff
@@ -0,0 +1,547 @@
+---
+id: ffbc54
+name: No keyboard shortcuts use only printable characters
+rule_type: atomic
+description: |
+  This rule checks that if keyboard shortcuts are implemented using only printable characters, then there is a mechanism to disable the shortcut, or to remap the shortcut to use one or more non-printable characters keys, or the shortcut for a user interface component is only available when that component has focus.
+accessibility_requirements:
+  wcag21:2.1.4: # Character Key Shortcuts (A)
+    forConformance: true
+    failed: not satisfied
+    passed: satisfied
+    inapplicable: further testing needed
+acknowledgements:
+  authors:
+    - João Vicente
+    - Carlos Duarte
+---
+
+## Applicability
+
+The rule applies to any [keyboard event][]: 
```

Comment:
Concerning composition and https://github.com/act-rules/act-rules.github.io/pull/873#discussion_r409389247 I think that the situation has changed because the rule now targets events and not "shorcuts". The example in that message was about an element that would react to several different events (with `key` 'a', 'b', and 'c'). And this shortcut wasn't passing the rule.

Now, the rule targets each of the 3 events (and also the events with `key` 'd', 'e', … that cause no change in content). So now we can have each of these events handled in a separate way, which is already what the expectation does, and a composite rule could make sense again.


OTOH, I think now that with the introduction of the 2 new definitions, the conditions are short enough and making this a composite rule wouldn't be that beneficial (wouldn't really help understanding it). So I guess that we are good for now… (might change if the expectations get too complex again 😉 )

Reply:
I agree with both statements :-)